src/glum/_glm.py

```diff
@@ -2251,12 +2266,23 @@ class GeneralizedLinearRegressor(GeneralizedLinearRegressorBase):
         If true, then the expected information matrix is computed by default.
         Only relevant when computing robust standard errors.
 
-    categorical_features : str, optional (default = "{name}[{category}]")
+    categorical_format : str, optional (default = "{name}[{category}]")
         Format string for categorical features. The format string should
         contain the placeholder ``{name}`` for the feature name and
         ``{category}`` for the category name. Only used if ``X`` is a pandas
         DataFrame.
 
+    cat_missing_method: str {'fail'|'zero'|'convert'}, default 'fail'
```

Comment:
```suggestion
    cat_missing_method: str {'fail'|'zero'|'convert'}, default='fail'
```

Reply:
We should unify the notation for default parameters in a different PR (not pertaining to your changes here). The notation with defaults in parenthesis does not seem to be an [officially recognized style](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) though.

Reply:
Agreed.